activesupport/lib/active_support/core_ext/erb/util.rb

```diff
@@ -17,34 +45,8 @@ module Util
     SAFE_XML_TAG_NAME_REGEXP = /\A[#{TAG_NAME_START_CODEPOINTS}][#{TAG_NAME_FOLLOWING_CODEPOINTS}]*\z/
     TAG_NAME_REPLACEMENT_CHAR = "_"
 
-    # A utility method for escaping HTML tag characters.
-    # This method is also aliased as <tt>h</tt>.
-    #
-    #   puts html_escape('is a > 0 & a < 10?')
-    #   # => is a &gt; 0 &amp; a &lt; 10?
-    def html_escape(s)
-      unwrapped_html_escape(s).html_safe
-    end
-
-    silence_redefinition_of_method :h
-    alias h html_escape
-
-    module_function :h
-
-    singleton_class.silence_redefinition_of_method :html_escape
-    module_function :html_escape
```

Comment:
Hum, `module_function` makes the instance method private right?

We might be missing a couple `private` calls to be 100% compatible.

Reply:
Made those methods private, and pushed https://github.com/rails/rails/pull/46573/commits/cedf12d579fbe441edbad33e3f94aba7a0762ddb

Reply:
:thinking: Looks like I need to figure out something for making it `private` correctly

Reply:
https://github.com/rails/rails/pull/46573/commits/d6c42b3fba456d1882682a5348610309164b1edd